sys/arm64/arm64/pmap.c

```diff
@@ -5661,6 +5665,18 @@ pmap_enter_l3c(pmap_t pmap, vm_offset_t va, pt_entry_t l3e, u_int flags,
 	KASSERT(!VA_IS_CLEANMAP(va) || (l3e & ATTR_SW_MANAGED) == 0,
 	    ("pmap_enter_l3c: managed mapping within the clean submap"));
 
+#if __has_feature(capabilities)
```

Comment:
Put it under #ifdef CHERI_CAPREVOKE instead?

Reply:
Yeah, that makes more sense.